### DISCUSSION TITLE
Get Most Recent Relation Item in 1 to M Relation

### DISCUSSION THREAD

**Discussion by jmickey** [prisma/prisma#6513] [Q&A] [answered]
Hi, I have a query where I want to get the most recently created item from a relation, is this the easiest way to achieve this?

```typescript
const userWithLatestPost = await prisma.user.findUnique({
  where: { id: 1 },
  include: {
    post: {
      orderBy: {
        createdAt: 'desc',
      },
      take: 1,
    },
  },
});
```

**Comment by ryands17** [comment, marked as answer]
@jmickey 👋 
Yes this will be the easiest way 👍

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
